pkg/k8s/watchers/namespace.go

```diff
@@ -24,12 +25,13 @@ import (
 func (k *K8sWatcher) namespacesInit() {
 	apiGroup := k8sAPIGroupNamespaceV1Core
 
-	synced := false
+	var synced atomic.Bool
+	synced.Store(false)
```

Comment:
this is probably redundant, zero value should be false

Reply:
Yeah, I opted for explicitness so that when someone fetches all references of the variable, they see an explicit "false" (next to an explicit "true"). It makes it slightly easier to tell how this variable is used.